sycl/doc/GettingStartedAlveo.md

```diff
@@ -0,0 +1,449 @@
+Getting started with SYCL with a Xilinx FPGA U200 Alveo board and Ubuntu 19.04
+==============================================================================
+
+We assume that you have the latest Ubuntu 19.04 version installed on
+an `x86_64` machine.
+
+It might work for some other version of Ubuntu or Debian with some
+adaptations.
+
+
+## Installing the Alveo U200 board
+
+If you do not have a real board and wants to use only software or
+hardware emulation, just skip this section.
+
+Install an Alveo U200 board in the machine with the right cooling.
+The PCIe auxiliary power is not necessary for simple test not using the full
+power of the board.
+
+
+### Use a modern BIOS
+
+Update the BIOS of your machine to the latest version. The Alveo U200
+board might not be detected by `lspci` at the PCIe level if you do not
+have a good BIOS.
+
+If you are running Linux on top of an EFI BIOS, you can probably use
+the firmware capsule concept and try:
+```bash
+# Install the firmware manager and daemon
+sudo apt install fwupdate fwupd
+# Refresh the list of available firmware
+sudo fwupdmgr refresh
+# Do the firmware update if any
+sudo fwupdmgr update
+```
+
+If you are not running an EFI BIOS, you can follow the manual recipe
+of the BIOS update for your the motherboard. Typically look for the
+latest BIOS version, put it on a FAT32-formatted USB stick and goes
+into the BIOS setup at boot time to ask for the explicit
+update. Often, there is no need to build a bootable USB stick.
+
+### Use the right Linux kernel
+
+To run on some real hardware, we need a Linux kernel supported by the
+very picky XRT Linux kernel driver, according to
+https://github.com/Xilinx/XRT/issues/1384
+
+So, create a `/etc/apt/sources.list.d/xrt-bionic.list` file with this content:
+```
+# Add old packages from Ubuntu 18.04 to have XRT kernel driver compiling
+deb http://us.archive.ubuntu.com/ubuntu/ bionic main restricted
+```
+
+Then install the less oldest supported kernel by XRT
+```bash
+sudo apt update
+sudo apt install linux-image-4.15.0-50-generic \
+  linux-modules-4.15.0-50-generic \
+  linux-modules-extra-4.15.0-50-generic \
+  linux-headers-4.15.0-50 linux-headers-4.15.0-50-generic
+```
+
+Edit `/etc/default/grub` to save the kernel choice at boot to be the
+default one by default:
```

Comment:
I'm not too sure the wording of this sentence is great, I understand this sentence but  "default one by default", doesn't seem to be the right choice of words, not too sure what it could be instead though (first default perhaps more explicit e.g. "the new kernel by default")